techniques/general/G70.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html><html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml"><head><title>Providing a function to search an online dictionary</title><link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"/></head><body><h1>Providing a function to search an online dictionary</h1><section class="meta"><p class="id">ID: G70</p><p class="technology">Technology: general</p><p class="type">Type: Technique</p></section><section id="applicability"><h2>When to Use</h2>
       <p>All technologies</p>
    </section><section id="description"><h2>Description</h2>
-      <p>The objective of this technique is to provide the definition of words, phrases, jargon, or abbreviation expansions by adding a mechanism to access an on-line dictionary to the web page. This technique uses existing resources on the Web to provide the definition rather than requiring the author to create a glossary or other mechanism within the site. By providing access from within the web page, a user can easily locate the desired definition. This technique can only be used if the online dictionary returns the correct definition.</p>
+      <p>The objective of this technique is to provide the definition of words, phrases, jargon, or abbreviation expansions by adding a mechanism to access an on-line dictionary to the web page. This technique uses existing resources on the web to provide the definition rather than requiring the author to create a glossary or other mechanism within the site. By providing access from within the web page, a user can easily locate the desired definition. This technique can only be used if the online dictionary returns the correct definition.</p>
```

Comment:
"... resources on the web to provide..."

This feels like it verges on being more like proper noun usage? 
> lowercase except if proper noun meaning World Wide Web